version4.go

```diff
@@ -30,6 +30,7 @@ func NewRandom() (UUID, error) {
 	return NewRandomFromReader(rander)
 }
 
+// NewRandomFromReader returns an UUID based on bytes read from given io.Reader.
```

Comment:
This should be "a UUID" as UUID is pronounced you-you.

Reply:
Done.